mailpoet/tests/integration/REST/Automation/Automations/AutomationPutEndpointTest.php

```diff
@@ -31,6 +31,29 @@ public function _before() {
     $this->assertInstanceOf(Automation::class, $this->automation);
   }
 
+  public function testEditorIsAllowed(): void {
+    wp_set_current_user($this->editorUserId);
+    $data = $this->put(
+      sprintf(self::ENDPOINT_PATH, $this->automation->getId()),
+      [
+        'json' => [
+          'name' => 'Test',
+        ],
+      ]
+    );
+
+    $this->assertNotSame([
+      'code' => 'rest_forbidden',
+      'message' => 'Sorry, you are not allowed to do that.',
+      'data' => ['status' => 401],
+    ], $data);
```

Comment:
This check doesn't seem to make much sense given the check just below (and in all of the other occurrences) but I guess no harm in keeping it either.